R/calcIOEdgeBuildings.R

```diff
@@ -36,7 +36,10 @@ calcIOEdgeBuildings <- function(subtype = c("output_EDGE", "output_EDGE_building
   # delete NAs rows
   ieamatch <- ieamatch[c("iea_product", "iea_flows", target, "Weight")] %>%
     na.omit() %>%
-    unite("target", all_of(target), sep = ".")
```

Comment:
this is only working with the new IEA data, right? Is there a version number which could be increased to indicate this?

Reply:
We are explicitly reading in new IEA data, so this only has to work with new IEA data, right?
https://github.com/fbenke-pik/mrcommons/blob/IEA/R/readIEA.R#L21

Reply:
yes, right, it is all in one PR but if we anyway have a version number, this would ensure consistency too

Reply:
@robinhasse once this mrrcommons branch is merged and we have a version number, you might have to update the required version of mrcommons here: https://github.com/pik-piam/mredgebuildings/blob/main/DESCRIPTION

Reply:
done

Reply:
Thanks for the hint. This is still under development and not in use currently but still good that you reminded me.